q2_qemistree/_itol_metadata.py

```diff
@@ -72,18 +83,45 @@ def get_itol_visualization(classified_feature_data: str,
         fh.write('LABELS\n'
                  'SEPARATOR TAB\n'
                  'DATA\n')
-        if ms2_label:
-            for idx in fdata.index:
-                ms2_compound = fdata.loc[idx, 'ms2_compound']
-                if pd.notna(ms2_compound) and not ms2_compound.isspace():
-                    label = ms2_compound
-                else:
-                    label = fdata.loc[idx, feature_data_column]
-                fh.write(idx + '\t' + label + '\n')
-        else:
-            for idx in fdata.index:
-                label = fdata.loc[idx, feature_data_column]
-                fh.write(idx + '\t' + label + '\n')
```

Comment:
@qiyunzhu We are missing this chunk of code from your PR as well? Is there a chance you need to update your branch to the current master and then update your PR?